hail/src/main/scala/is/hail/cxx/Emit.scala

```diff
@@ -355,9 +556,9 @@ class Emitter(fb: FunctionBuilder, nSpecialArgs: Int) { outer =>
                  |""".stripMargin)
 
           case None =>
-            val xs = genSym("xs")
-            val ms = genSym("ms")
-            val i = genSym("i")
+            val xs = fb.genSym("xs")
```

Comment:
would kind of prefer that these also be variables, but that's not a change you made here and we can clean up later.